python/python/glide/async_commands/cluster_commands.py

```diff
@@ -415,6 +415,41 @@ async def function_delete(
             ),
         )
 
+    async def fcall_ro_route(
+        self,
+        function: str,
+        arguments: Optional[List[str]] = None,
+        route: Optional[Route] = None,
+    ) -> TClusterResponse[TResult]:
```

Comment:
Is the return type supposed to be `TClusterResponse[TResult]` or `TClusterResponse[Optional[TResult]]`? I thought the Java implementation was nullable so we'd need to use `Optional` here, but I dunno.

Reply:
From my understanding, I don't think it is nullable. @Yury-Fridlyand  can you please help answer this?

Reply:
I have updated the return type to Optional.